app/infrastructure/sql_handler.go

```diff
@@ -16,8 +17,10 @@ type SqlHandler struct {
 
 func NewSqlHandler() database.SqlHandler {
 	err := godotenv.Load()
+	if err != nil {
+		fmt.Println("No .env file available")
```

Comment:
Is it OK to show this info only in console? I think error should be called.And inside the error, you should create comment.
Or use the error handler line 24 is better. Maybe we don't need to call error handler twice because we don't need to separate these two errors.

Reply:
in this case, actually throwing an error is not a good idea.
since dotenv is only for development mode on our local pc's.

in case of heroku or aws, dotenv will ALWAYS be throwing an error.
(that is what was breaking my previous deployments).

So in this case, we can either leave it just a message (which will always occur on deployment).
OR we could add an APP_ENV enviromental varaible..
and set it to "development" on PC, and "production" on the server..
and have an if clause here..
IF "development" && dotenv !=nil throw error, else show message